examples/src/models/embeddings/hf_transformers.ts

```diff
@@ -0,0 +1,14 @@
+import { HuggingFaceTransformersEmbeddings } from "langchain/embeddings/hf_transformers";
+
+const model = new HuggingFaceTransformersEmbeddings({
+  modelName: "Xenova/all-MiniLM-L6-v2", // In Node.js defaults to process.env.OPENAI_API_KEY
```

Comment:
I wouldn't say that this defaults to `OPENAI_API_KEY` but I guess this is generated by some automation tool

Reply:
Oh whoops thanks - was from the original PR